### PR TITLE
[Revamp] Migrate to slalpha5 and add new operations

### PR DESCRIPTION
**Pull request by Chuhaa**
## Purpose

- Migrate existing connector to slbeta1.
- Add new operations.
- Add display annotation.
- Add samples.
- Update documents.
- Update CI/CD pipelines.

## Goals
Fixes https://github.com/wso2-enterprise/choreo/issues/4735
Fixes https://github.com/wso2-enterprise/choreo/issues/6214

## Documentation
Please find the design document here https://docs.google.com/document/d/1JWBPeqmihNT-0R8HPbAevrBkWFdVACNw40xNRTC-P0M/edit?usp=sharing

## Automation tests
 - Unit tests 
    Tested
 - Integration tests
    Tested

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Samples
```
import ballerina/log;
import ballerinax/microsoft.excel;

configurable string clientId = ?;
configurable string clientSecret = ?;
configurable string refreshToken = ?;
configurable string refreshUrl = ?;
configurable string workbookIdOrPath = ?;

excel:ExcelConfiguration configuration = {
    authConfig: {
        clientId: clientId,
        clientSecret: clientSecret,
        refreshToken: refreshToken,
        refreshUrl: refreshUrl
    }
};

excel:Client excelClient = check new (configuration);

public function main() {
    excel:Worksheet|error response = excelClient->addWorksheet(workbookIdOrPath, "sheetName");
    if (response is excel:Worksheet) {
        log:printInfo(response.toString());
    }
}
```

## Test environment
SLAlpha 5
JDK 11
Ubuntu 20.04

### CODE REVIEW

File: excel/client.bal [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+# Microsoft Excel connector client endpoint
+public client class Client {
```
Comment (LakshanSS) [inline excel/client.bal:20]: Aren't we gonna add display annotation for client like this?
https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/8ad5e06895588ec206de7b60228a154ed2dd0562/calendar/endpoints.bal#L23

File: excel/tests/test.bal [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+        clientSecret: clientSecret,
+        refreshToken: refreshToken,
+        refreshUrl: refreshUrl
+    }
+};
+
+Client excelClient = check new (configuration);
+string workBookId = workbookIdOrPath;
+string worksheetName = "testSheet";
+string tableName = "";
```
Comment (LakshanSS) [inline excel/tests/test.bal:40]: In constants.bal, there is already constant defined as `EMPTY_STRING `
But in this test.bal, still `""` is used in many places.

Shall we use `EMPTY_STRING ` in all places?

Comment (Chuhaa) [reply excel/tests/test.bal]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/commit/5faa81d3bd0c91f10272734f80b7d29c9ee53f63

File: excel/client.bal (hunk 1 above)
Comment (Chuhaa) [reply excel/client.bal:21]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/commit/5faa81d3bd0c91f10272734f80b7d29c9ee53f63

File: .github/workflows/pull-request.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ballerina Build
+        uses: ballerina-platform/ballerina-action/@slalpha5
+        with:
+          args:
+            build -c --skip-tests
```
Comment (LakshanSS) [inline .github/workflows/pull-request.yml:16]: Why are we skipping tests? Its better to run tests after PR is merged right?

Comment (Chuhaa) [reply .github/workflows/pull-request.yml:16]: After PR merged run will be considered as on push job. Here it's only for creating a PR.

**Pull request merged**